src/applications/facility-locator/config.js

```diff
@@ -195,3 +196,16 @@ export const facilityTypesOptions = {
   [LocationType.CEMETARY]: 'VA cemeteries',
   [LocationType.VET_CENTER]: 'Vet Centers',
 };
+
+export const radiusFromBoundingBox = fbox => {
+  let radius = distBetween(
+    fbox[0].bbox[1],
+    fbox[0].bbox[0],
+    fbox[0].bbox[3],
+    fbox[0].bbox[2],
+  );
+
+  if (radius < MIN_RADIUS) radius = MIN_RADIUS;
+
+  return radius;
+};
```

Comment:
Seems like this function belongs in `utils/facilityDistance.js`?

Reply:
Sounds good